app/uk/gov/hmrc/pillar2submissionapi/services/SubmitUKTRService.scala

```diff
@@ -38,16 +39,20 @@ class SubmitUKTRService @Inject() (pillar2Connector: UKTaxReturnConnector)(impli
         response.json.validate[UKTRSubmitSuccessResponse] match {
           case JsSuccess(success, _) => success
           case JsError(errors) =>
+            logger.error(s"Got an error while parsing ${response.json}")
+            // change response here. We should not return this error to third parties
```

Comment:
is this comment to be actioned in this ticket?

Reply:
this will have to be a subsequent PR. I've noticed these errors being returned in multiple places (for BTN too)